patches/server/0004-Paper-config-files.patch

```diff
@@ -1105,6 +1106,7 @@ index 0000000000000000000000000000000000000000..fa191f418079a8ee24326c5952d12d44
 +                    .register(DoubleOrDefault.SERIALIZER)
 +                    .register(BooleanOrDefault.SERIALIZER)
 +                    .register(Duration.SERIALIZER)
++                    .register(ChunkPacketBlockControllerAntiXray.EngineMode.SERIALIZER)
```

Comment:
The reason I added this in the X-ray patch, is that now the server won't compile if you only some patches before X-ray.

Reply:
As discussed on Discord in #paper-contrib I have moved the `EngineMode` stuff to the config patch.